osu.Game.Rulesets.Osu/Difficulty/Skills/OsuStrainSkill.cs

```diff
@@ -28,6 +28,8 @@ public abstract class OsuStrainSkill : StrainSkill
         /// </summary>
         protected virtual double DifficultyMultiplier => 1.06;
 
+        protected List<double> objectStrains = new List<double>();
```

Comment:
I feel like we should do what we discussed in #16260 and #16122 first to store per-skill difficulty in objects properly

Reply:
for the record: we've agreed that we can refactor after the merge once we're ready, so that we can move forward and not be stuck here for months